app/assets/index.html

```diff
@@ -20,5 +20,6 @@
 </div>
 
 <script src="scripts/main.js"></script>
+<script>require('app/scripts/main');</script>
```

Comment:
`modules.js` would include all your application files (all the JS in `app/`), except for the `main` module. It would require a little more configuration to do this, but perhaps just one line.